src/data/actions/authentication.js

```diff
@@ -48,14 +51,45 @@ const logoutEvent = () => ({
   type: LOGOUT,
 });
 
-const logout = () => (
-  (dispatch) => {
-    const cookies = new Cookies();
-    cookies.remove(cookieName);
-    dispatch(logoutEvent());
+const logout = () => ((dispatch) => {
+  const cookies = new Cookies();
+  cookies.remove(cookieName, {
+    path: '/',
   });
+  dispatch(logoutEvent());
+});
+
+// When loading user details from the JWT stored on the local browser
+const localUserLoaded = email => ({
+  type: LOCAL_USER_LOADED,
+  payload: { email },
+});
+
+// This is used when the local browser does not have a JWT with the user details. This is
+// not an error condition, the user may not have logged in yet. We still capture this event
+// for tracking purposes.
+const localUserMissing = () => ({
```

Comment:
I'm not sure this is necessary, since it doesn't change the store, but it seemed like a useful event to capture

Reply:
i'm ok with leaving it in